core/src/main/scala/chisel3/Mem.scala

```diff
@@ -392,4 +392,153 @@ sealed class SyncReadMem[T <: Data] private[chisel3] (
   }
   // note: we implement do_read(addr) for SyncReadMem in terms of do_read(addr, en) in order to ensure that
   //       `mem.read(addr)` will always behave the same as `mem.read(addr, true.B)`
+
+  /** Creates a read-write accessor into this SyncReadMem.
+    *
+    * @param idx memory element index to write into
+    * @param writeData new data to write
+    * @param enable enables access to the memory
+    * @param isWrite when enable is true, if this access is a write
+    *
+    * @return the memory port accessor
```

Comment:
what is "the memory port accessor"? Doesn't this just return the read data...? So should this be @return the read data, defined only when en is true and isWrite is false?